modules/dnn_superres/src/dnn_superres.cpp

```diff
@@ -0,0 +1,385 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+
+#include "opencv2/dnn_superres.hpp"
+
+namespace cv
+{
+    namespace dnn_superres
+    {
+
+        int DnnSuperResImpl::layer_loaded = 0;
```

Comment:
Please avoid unnecessary indentation in namespaces.

Reply:
Done, I reformatted the code.